### ISSUE TITLE
Add support for `vulcanSmart` type

### ISSUE THREAD

**Issue by thecheatah** [vojtamolda/homebridge-ecobee3-sensors#30]
I have the latest Ecobee and needed to add the `vulcanSmart` model to get it to work

Added the following to platform.js
```
  for (var thermostatConfig of reply.thermostatList) {                                                                                                                                                                                                                     
    if ((thermostatConfig.modelNumber != 'vulcanSmart') && (thermostatConfig.modelNumber != 'athenaSmart') && (thermostatConfig.modelNumber != 'apolloSmart') && (thermostatConfig.modelNumber != 'nikeSmart')) {                                                          
      this.log.info("Not supported thermostat | " + thermostatConfig.name + " (" + thermostatConfig.modelNumber + ")");                                                                                                                                                    
      continue                                                                                                                                                                                                                                                             
    
```

**Comment by vojtamolda** [comment]
Hello @thecheatah,

Thanks a lot for sharing your patch. The change looks great. Could you, please, submit it as a pull-request here?

Thanks and have a good one!

  Vojta

**Comment by thecheatah** [comment]
@vojtamolda added pull request here: https://github.com/vojtamolda/homebridge-ecobee3-sensors/pull/31

**Comment by vojtamolda** [comment]
Thanks for the PR! I have just published [version 0.1.7 to NPM](https://www.npmjs.com/package/homebridge-ecobee3-sensors).
